moderatorguide.md

```diff
@@ -3,62 +3,62 @@
 ## Your tasks
  
 
-* open the program by introducing the event, organizers and the (first) speaker,
+* open the program by introducing the event, organizers and the (first) speaker
 * remind the audience of the event format and explain how and when to ask questions
-* monitor the participant list for raised hands and enable those attendees to ask questions,
-* handle any technical issues with the execution of the virtual conference,
-* monitor participant behavior in the chat for breaches of the code of conduct,
-* give a closing remarks.
+* monitor the participant list for raised hands and enable those attendees to ask questions
+* handle any technical issues with the execution of the virtual conference
+* monitor participant behavior in the chat for breaches of the code of conduct
+* give a closing remarks
 
-Below we provide more detailed information on certain aspects of the moderator role. 
+Below we provide more detailed information on certain aspects of the moderator role. Note that the moderator role requires full concentration, lot of focus and continuous monitoring of multiple channels. In that sense, moderating an online event is significantly more demanding than the role of a chair in an offline conference.
 
 ## How to
 
 ### Opening and closing remarks
 
 At the beginning of the seminar: 
 
-* highlight the speaker’s screen-share,
-* introduce the speaker (name, current institution, something about the talk or their current work),
-* unmute the speaker and invite them to begin their seminar.
+* ensure that everyone is muted and cannot unmute themselves by clicking "mute all participants" and unchecking "participants may unmute themselves" checkbox
+* announce that the talk is recorded and start the recording
+* deliver the opening remarks, introduce VSF and the speaker
+* turn on spotlight of the speaker’s webcam stream
+* unmute the speaker and invite them to begin their seminar
 
 At the end of the seminar: 
 
 * thank the speaker 
 * initiate the post-talk question/answer period
-* stop highlighting speaker's screen-share.
+* stop highlighting speaker's screen-share
 
-NB: a virtual conference cannot have applause after the seminar ends! However, Zoom does have a clapping hands image that audience members may put up next to their name in the participants list. 
+NB: a virtual conference cannot have applause after the seminar ends! However, you may invite the audience to use for example the Zoom "applaud" emote.
 
```

Comment:
At the bottom we ask moderators to encourage feedback from the audience on technical issues. We should therefore include a bullet point somewhere about that. Perhaps this makes sense at before the seminar, since the technical issues may disrupt or cancel the seminar and therefore waiting until the end for this encouragement may be less productive.